src/components/dashboard/dashboard.component.tsx

```diff
@@ -17,12 +18,70 @@ import {
   Tag,
 } from "carbon-components-react";
 import styles from "./dashboard.css";
-import { Download, Edit } from "@carbon/icons-react/next";
+import { Download, Edit, DocumentImport } from "@carbon/icons-react/next";
 import { useTranslation } from "react-i18next";
 
+function getTableRows(t) {
+  const tableRows = [];
+  const { data } = GetPocForms();
+  data?.forEach((row, key) =>
```

Comment:
const tableRows = data?.map((row,key) => {return ....})

No need to use forEach, you can use the `map`